src/NuGet.Clients/NuGet.Tools/NuGetPackage.cs

```diff
@@ -189,7 +189,7 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
 
             await NuGetBrokeredServiceFactory.ProfferServicesAsync(this);
 
-            VsShellUtilities.ShutdownToken.Register(RegisterEmitVSInstancePowerShellTelemetry);
+            VsShellUtilities.ShutdownToken.Register(InstanceCloseEvent.OnShutdown);
```

Comment:
Did you validate using telemetry monitor that current event is emitted?

Reply:
Yes. My first attempt got some of the existing PMC properties wrong 😳 But I sorted that out, so it seems good to me now.